dataland-frontend/src/components/forms/parts/elements/derived/ProcurementCategoryFormElement.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div class="form-field">
+    <div data-test="dataPointToggle" class="form-field border-none vertical-middle">
+      <InputSwitch
+        data-test="dataPointToggleButton"
+        inputId="dataPointIsAvailableSwitch"
+        @click="this.isItActive = false"
+        v-model="isItActive"
+      />
+      <h5 data-test="dataPointToggleTitle" class="m-2">
+        {{ label }}
+      </h5>
+    </div>
+    <FormKit type="group" :name="name" v-if="isItActive">
+      <div data-test="ProcurementCategoryFormElementContent">
+        <div class="form-field border-none">
+          <NaceCodeFormFieldBindData
+            label="Definition Product Type/Service"
+            description="Define the procured product types/services per category (own operations)"
+            name="procuredProductTypesAndServicesNaceCodes"
+            v-model:selectedNaceCodesBind="procuredProductTypesAndServicesNaceCodesValue"
+          ></NaceCodeFormFieldBindData>
+        </div>
+
+        <div class="form-field border-none">
+          <UploadFormHeader
+            label="Order Volume per Procurement %"
+            description="State your order volume per procurement category in the last fiscal year (percentage of total volume) (own operations)"
+            :is-required="false"
+          />
+          <FormKit
+            type="text"
+            v-model="percentageOfTotalProcurementValue"
+            name="percentageOfTotalProcurement"
+            validation="number"
+            inner-class="long"
+          />
+        </div>
+        <div class="form-field border-none">
+          <MultiSelectFormFieldBindData
+            label="Sourcing Country per Category"
+            placeholder="Countries"
+            description="Name the sourcing countries per category (own operations)"
+            name="suppliersPerCountryCode"
+            :options="allCountry"
+            optionLabel="label"
+            :optionValue="false"
+            v-model:selectedItemsBindInternal="selectedCountries"
+            innerClass="long"
+            :ignore="true"
+          />
+        </div>
+        <div class="form-field border-none">
+          <div class="flex justify-content-between">
+            <UploadFormHeader
+              v-if="selectedCountries.length > 0"
+              label="Number of Direct Supliers"
```

Comment:
Why is the label value hardcoded, if we can get it from the `DataModelsTranslations.ts` ?

Reply:
I was just about to ask a similar question. I'll change this.

Reply:
Done as discussed.

Reply:
Reminder to myself to check if we should use the translations-file for other labels too.

Reply:
Ok I made up my mind.  I suggest leaving the hardcoded labels here, since there are too many other labels that are hardcoded too.
Let's stay consistent.

Could you revert this change here and write "Number of Direct Suppliers" hardcoded into the `label` here again please?

Reply:
Done.

Reply:
👍